lib/index.js

```diff
@@ -28,50 +34,35 @@ function addRoutes(app) {
   app.post(cfg.routes.verify, brRest.when.prefers.ld,
     asyncHandler(async (req, res) => {
       const {challenge, domain, presentation} = req.body;
-      let result;
+      let _report;
       try {
-        result = await vc.verifyPresentation({
+        _report = await vc.verify({
           challenge,
           documentLoader: loader.documentLoader,
           domain,
+          purpose: new AuthenticationProofPurpose({challenge, domain}),
```

Comment:
This is confusing ... why are `challenge` and `domain` being passed to `vc.verify` if they are also being passed to `purpose` here? Can we just remove the `purpose` param and use the default?

Reply:
This is confusing: I removed the purpose and it fails. Looking into it.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-vc-verifier/pull/3/commits/40c12f89ebd08c7dd8bc6c479526dbfbb71e72ef

Do we need an issue in vc-js so you can specify a purpose for credentials?